netflix/src/components/main/sort/GenreOption.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { setCurrentGenre, setFilms } from "../../../redux/films";
+import { useDispatch } from "react-redux";
+import { filterGenre } from "../../../redux/api";
+
+const GenreOption = (props: any) => {
```

Comment:
Yo my knowledge, usually preferred way to work with redux is using `connect()`
https://react-redux.js.org/api/connect

Reply:
So just familarize yourself with it